examples/Makefile

```diff
@@ -32,7 +32,7 @@ GOVETTARGETS=cmd \
 	pkg
 
 ORG=networkservicemesh
-BUILD_CONTAINERS=icmp-responder-nse nsc
+BUILD_CONTAINERS=icmp-responder-nse nsc nsm-generate-sriov-configmap
```

Comment:
Maybe not on this PR, but lets look at pulling this up into the new top level build machinery, and giving it its own circle build-* job.